examples/unitTestExample/README.md

```diff
@@ -0,0 +1,4 @@
+# Setup
+
+Tests can be run via the command line without fail, however, to get them to run in Android Studio you will need to perform the following steps in your run configurations in Android Studio:
+![Unit Test Config Settings](unittest-config-settings.png)
```

Comment:
What part of the setup is this needed for? I don't think I ever had to do this, for neither Junit3 or 4?